Examples/Framework/include/ActsExamples/EventData/ModuleCluster.hpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Digitization/Segmentation.hpp"
+#include "ActsExamples/EventData/IndexSourceLink.hpp"
+#include "ActsExamples/EventData/Measurement.hpp"
+
+#include <cassert>
+#include <vector>
+
+namespace ActsExamples {
```

Comment:
Is there a corresponding class in Core or should customers create their own version of this in their code?